### DISCUSSION TITLE
Why does the `remember_me` cookie value contain the entity class name?

### DISCUSSION THREAD

**Discussion by ToshY** [symfony/symfony#53334] [Q&A] [answered]
I was looking at the `remember_me` cookie in the chrome dev console and saw the following:

```text
App.Entity.Site.SiteUser:WH9azJl~:1735569643:<string>-<string>~<string>_<string>~
```

Why is `App.Entity.Site.SiteUser` in the cookie name? Doesn't seem that I'm able to change the name from the config as it's using the `$user::class` for this in the [`PersistentRememberMeHandler`](https://github.com/symfony/security-http/blob/acc9931d75cd16de08b1663223cb8ab36f61cc0c/RememberMe/PersistentRememberMeHandler.php#L54).

For some reason I'm not really "comfortable" to exposing backend entity class name to the frontend. I'm not sure if my concern is justified or not, but I would like to know what others think / why this is implemented this way.

**Versions**
Symfony: `6.4.x`

**Comment by MatTheCat** [comment, marked as answer]
See #42349

**Reply by ToshY** [reply]
Hey @MatTheCat 👋 

Thanks, that explains it.
